inventory_management/inventory_management/report/stock_balance/stock_balance.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2024, Viny Selopal and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe import _
+
+
+def execute(filters=None):
+    columns = get_columns()
+    data = get_data(filters)
+
+    return columns, data
+
+
+def get_columns():
+    columns = [
```

Comment:
Use tabs everywhere. Some files also have mixed tabs and spaces
https://github.com/frappe/erpnext/wiki/Coding-Standards#use-simple-indenting